client/src/main/scala/drt/client/components/ContactPage.scala

```diff
@@ -185,7 +186,7 @@ object FeedBackComponent {
                         "border" -> "1px solid white",
                         "color" -> "white",
                         "font-weight" -> "bold",
-                        "font-size" -> "1.50rem")))(
+                        "font-size" -> buttonTheme.typography.fontSize)))(
```

Comment:
Dude themes work by wrapping the components so that you don't then need specify the styles directly on the component